### PR TITLE
[Enhancement] add switch of quantile_state column

### PR DESCRIPTION
**Pull request by spaces-X**
# Proposed changes

Add switch for quantile_state column, default false.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (Yes)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by spaces-X** [comment]
@wangbo  Add a switch of `quantile_state` column to ensure stability, and it will be removed after subsequent tests are stable

**Review by wangbo — DISMISSED** [review]
LTGM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java [hunk 1]
```diff
@@ -165,6 +166,10 @@ public void analyze(boolean isOlap) throws AnalysisException {
 
         Type type = typeDef.getType();
 
+        if(!Config.enable_quantile_state_type && type.isQuantileStateType()) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java:169]: Make the error msg more clear. You can refer to `enable_create_sync_job`

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 2]
```diff
@@ -1644,4 +1644,8 @@
      */
     @ConfField(mutable = true)
     public static boolean skip_compaction_slower_replica = true;
+
+    @ConfField(mutable = true)
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1648]: Add comment for it.
And it is `masterOnly`

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**